app/permissions/permissions.ts

```diff
@@ -0,0 +1,30 @@
+import { Acl } from "@holoyan/adonisjs-permissions";
+
+export const create = await Acl.permission().create({
+  slug: "create",
+  title: "Create some resource",
+});
+
+export const update = await Acl.permission().create({
+  slug: "update",
+});
+
+export const read = await Acl.permission().create({
+  slug: "read",
+});
+
+export const destroy = await Acl.permission().create({
+  slug: "destroy",
+});
+
+export const approveDraft = await Acl.permission().create({
+  slug: "approve_draft",
+});
```

Comment:
this permission appears to be unused, as you require `solvro_admin` everywhere to approve drafts